integration/aggregate_stats_test.go

```diff
@@ -44,6 +46,7 @@ func TestAggregateCollStats(t *testing.T) {
 
 	var res []bson.D
 	err = cursor.All(ctx, &res)
+	require.NoError(t, cursor.Close(ctx))
```

Comment:
`cursor.All` (and some other helpers like `FetchAll`) always closes the cursor. I don't think we should add `cursor.Close`anywhere.

Reply:
Great! Let's just use helper here.